### PR TITLE
Release v1.27.0

### PR DESCRIPTION
**Pull request by AndrewSirenko**
**Is this a bug fix or adding new feature?**
Release

**What is this PR about? / Why do we need it?**
Releases v1.27.0 

**What testing is done?** 
CI 

- [x] Update changelog.md
- [x]  readme.md latest version and install table
- [x]  Makefile VERSION 
- [x] install.md all occurences of old version
- [x] charts/aws-ebs-csi-driver/Chart.yaml with new version & appVersion 
- [x] charts/aws-ebs-csi-driver/CHANGELOG.md with Helm changelog 
- [x] Update overlays by running make generate-kustomize

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
+# v1.27.0
+### Notable Changes
+* Pre-stop Lifecycle Hook enhancements ([#1895](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1895), [@torredil](https://github.com/torredil))
```
Comment (torredil) [inline CHANGELOG.md:3]: Suggest moving this entry down to Improvements section

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+
+### Bug Fixes
+* Fix taint removal retry for non-swallowed errors ([#1898](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1898), [@ConnorJC3](https://github.com/ConnorJC3))
+
+### Improvements
+* Remember likely bad device names for an hour ([#1889](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1889), [@ConnorJC3](https://github.com/ConnorJC3))
+* Use lsblk to safeguard against outdated symlinks ([#1878](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1878), [@ConnorJC3](https://github.com/ConnorJC3))
+* Remove premature CreateVolume error if requested IOPS is below minimum ([#1883](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1883), [@AndrewSirenko](https://github.com/AndrewSirenko))
```
Comment (torredil) [inline CHANGELOG.md:11]: Suggest bumping up this entry to Notable Changes section

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,16 @@
+# v1.27.0
+### Notable Changes
+* Pre-stop Lifecycle Hook enhancements ([#1895](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1895), [@torredil](https://github.com/torredil))
+
+### Bug Fixes
+* Fix taint removal retry for non-swallowed errors ([#1898](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1898), [@ConnorJC3](https://github.com/ConnorJC3))
+
+### Improvements
+* Remember likely bad device names for an hour ([#1889](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1889), [@ConnorJC3](https://github.com/ConnorJC3))
```
Comment (ConnorJC3) [inline CHANGELOG.md:9]: I know what you wrote is the name of the PR, but can you add something like `(Enable use of driver on AMIs with instance store mounts)`, as that's the customer facing change of this PR

**Review by ConnorJC3 — COMMENTED** [review]
Largely lgtm, agree with @torredil 's 2 comments

**Pull request merged**